ha/cli/cortxha.py

```diff
@@ -87,7 +87,7 @@ def command(self):
                 argParser.print_help(sys.stderr)
             else:
                 output = Output()
-                cluster = CortxClusterManager()
+                cluster = PcsClusterManager()
```

Comment:
This is fine for M0 but we should be setting the clusterManager based on values present in config file.

Reply:
Done